### ISSUE TITLE
Limit the loss when force sell at the market price is enabled

### ISSUE THREAD

**Issue by peweb** [chrisleekr/binance-trading-bot#531]
### Is your proposal related to a problem?

There is a setting to enable 

Force sell at the market price when recommendation is Strong sell and the profit is between 0 to trigger price

If enabled, the bot will use TradingView recommendation to sell the coin at the market price if the profit is over 0 but under the trigger price. When the condition is met and the TradingView recommendation is `Strong sell`, then the bot will place a market sell order immediately. If the auto-buy trigger is enabled, then it will place a buy order later. 

![image](https://user-images.githubusercontent.com/36623351/197567393-33fde5e6-dbda-45b1-9d85-2857622b6bbb.png)

### Describe the solution you'd like

Can we set it up to sell if the proffit is between X (>0) because currently this action can cause loss if the profit is less than commission. 
In general it would be great if the bot doesnt sell at a loss if possible:

![image](https://user-images.githubusercontent.com/36623351/197567961-9e5fa5df-a129-4b5a-89fb-ca5ec2920b45.png)

I think if there is a mimnal proffit above ZERO this can be minimized.

**Comment by dasbts** [comment]
I agree, this would be a good thing to implement. I have quite limited time these days otherwise I would look into the code for a PR, but spontaneously, add an if statement to the code where the param if this config setting is enabled exists, and if > X then go through with the sell, if minimum profit is false then don't care for minimum profit, and add another param to the config file.

**Comment by chrisleekr** [comment]
@peweb Sounds good feature to add.

I am currently refactoring TradingView. 
But I won't add this change at this time. 
Already have lots of changes.

**Comment by chrisleekr** [comment]
Closing this as a refactored v1 has been released. It's no longer relevant to the latest code.
